terraform/helm/fullnode/templates/restore.yaml

```diff
@@ -45,7 +45,7 @@ spec:
           value: {{ include "backup.loggingAddress" $ | quote }}
         {{- if (include "backup.pushGateway" $) }}
         - name: PUSH_METRICS_ENDPOINT
-          value: "{{- include "backup.pushGateway" $ }}/metrics/job/{{- .jobName | default "db_restore" }}"
+          value: "{{- include "backup.pushGateway" $ }}/api/v1/import/prometheus?extra_label=role={{- .jobName | default "db_restore" }}"
```

Comment:
nit: rename `pushGateway` to something like `pushMetricsEndpoint`. pushGateway might be easy to confuse with Prometheus PushGateway, whereas this. can now support anything that accepts Prometheus exposition format